construction_progress_management/data/ir_ui_view.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="sale_order_list_view" model="ir.ui.view">
+        <field name="name">Construction list view for Sale Order Lines</field>
+        <field name="inherit_id" ref="sale.view_order_line_tree"/>
+        <field name="mode">extension</field>
+        <field name="model">sale.order.line</field>
+        <field name="priority">1000</field>
+        <field name="type">list</field>
+        <field name="active" eval="True"/>
+        <field name="arch" type="xml">
+            <xpath expr="/list" position="replace">
+                <list default_order="order_id desc,sequence" editable="bottom" expand="True" decoration-info="display_type" decoration-it="display_type == 'line_note'" decoration-bf="display_type == 'line_section'">
+                    <header>
+                        <button class="btn btn-primary" name="%(action_sol_update_qty_delivered)d" type="action" string="Update Delivery"/>
+                    </header>
+                    <field name="order_id"                                              widget="many2one"     readonly="True"/>
+                    <field name="name"                                                                        readonly="True"/>
+                    <field name="product_uom_qty"                                       string="Qty"          readonly="True"                                                                                                              invisible="display_type"/>
+                    <field name="product_uom_id" groups="uom.group_uom"                 widget="many2one_uom" readonly="True"                                                                                                              invisible="display_type"/>
+                    <field name="x_delivered_progress"                                  widget="progressbar"  readonly="True"                                                                                                              invisible="x_delivered_progress == 0"/>
+                    <field name="x_quantity_increment"                    width="130px"                       readonly="display_type or not (is_service and x_service_policy == 'delivered_manual')"                                       invisible="(not display_type and not (is_service and x_service_policy == 'delivered_manual')) or display_type"/>
```

Comment:
You seem to repeat a lot the same condition. I'd create a non-store read-only computed field which contains the value of it, for readability.